assets/apis/example_SingleDefinition.json

```diff
@@ -24,6 +24,13 @@
             }
         }
     ],
+    "circuit_breaker":{
```

Comment:
Shall we make it a plugin, instead of changing proxy definition?

Reply:
it needs to be part of the definition because we use it on the transport layer :/

Reply:
we started as a plugin, but realised it really should be part of the definition